src/Solnet.Programs/SystemProgramData.cs

```diff
@@ -1,3 +1,4 @@
+using Newtonsoft.Json;

```

Comment:
Is this really necessary? We've been trying to minimize dependencies as much as possible

Reply:
Gone!